.github/workflows/pulp_images.yml

```
@@ -242,7 +242,11 @@ jobs:
             # Reuse the folders from the s6 mode tests
             FILE="docker-compose.folders.yml"
             # We'll pull the web image from a registry since we didn't build it.
-            WEB_TAG="${APP_BRANCH}"
+            if [ "${{ matrix.image_variant }}" == "nightly" ]
```

I think we're missing a `then` by the end of this line, like this:
```suggestion
            if [ "${{ matrix.image_variant }}" == "nightly" ] then
```

---

You forgot the semicolon.

Fixed it :)